stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/login/LoginFragment.kt

```diff
@@ -42,7 +43,10 @@ class LoginFragment : Fragment(R.layout.fragment_login) {
             }
         )
 
-        activity?.intent?.getStringExtra(EXTRA_CHANNEL_ID)?.let { cid ->
+        activity?.intent?.apply {
+            val channelId = getStringExtra(EXTRA_CHANNEL_ID)
+            val channelType = getStringExtra(EXTRA_CHANNEL_TYPE)
+            val cid = "$channelType:$channelId"
             viewModel.targetChannelDataReceived(cid)
         }
```

Comment:
The previous code included a null check on the return value of `getStringExtra`, and only executed code if that was non-null. We might want to have that check here as well, for the cases where Intent is non-null, but it doesn't have these values in it.